packages/app-runtime/test/modules/IdentityDeletionProcessStatusChangedModule.test.ts

```diff
@@ -48,4 +64,51 @@ describe("IdentityDeletionProcessStatusChanged", function () {
         const account = await runtime.multiAccountController.getAccount(CoreId.from(session.account.id));
         expect(account.deletionDate).toBeUndefined();
     });
+
+    describe("multi device", function () {
+        beforeAll(async function () {
+            runtimeDevice2 = await TestUtil.createRuntime(undefined, undefined, eventBus);
+            await runtimeDevice2.start();
+
+            const createDeviceResult = await session.transportServices.devices.createDevice({ name: "test", isAdmin: true });
+            const onboardingInfoResult = await session.transportServices.devices.getDeviceOnboardingInfo({ id: createDeviceResult.value.id, profileName: "Test" });
+            const localAccountDevice2 = await runtimeDevice2.accountServices.onboardAccount(onboardingInfoResult.value);
+            sessionDevice2 = await runtimeDevice2.selectAccount(localAccountDevice2.id.toString());
+
+            await session.transportServices.account.syncDatawallet();
+            await sessionDevice2.transportServices.account.syncDatawallet();
+        });
+
+        test("should set the deletionDate on the LocalAccount on a second device when an IdentityDeletionProcess is initiated", async function () {
+            const initiateDeletionResult = await session.transportServices.identityDeletionProcesses.initiateIdentityDeletionProcess();
+            expect(sessionDevice2!.account.deletionDate).toBeUndefined();
+
+            await sessionDevice2!.transportServices.account.syncDatawallet();
+
+            await expect(eventBus).toHavePublished(LocalAccountDeletionDateChangedEvent, (e) => e.data.deletionDate! === initiateDeletionResult.value.gracePeriodEndsAt!);
+
+            expect(sessionDevice2!.account.deletionDate!.toString()).toStrictEqual(initiateDeletionResult.value.gracePeriodEndsAt);
+
+            const account = await runtimeDevice2!.multiAccountController.getAccount(CoreId.from(sessionDevice2!.account.id));
+            expect(account.deletionDate!.toString()).toBe(initiateDeletionResult.value.gracePeriodEndsAt!.toString());
+        });
+
+        test("should unset the deletionDate on the LocalAccount on a second device when an IdentityDeletionProcess is cancelled", async function () {
```

Comment:
I would also adapt the test name to "...deletionDate of the LocalAccount..." instead of "...deletionDate on the LocalAccount...", as you did with the tests above.